File: 백준/Bronze/17202. 핸드폰 번호 궁합/핸드폰 번호 궁합.py
import sys
input=lambda:sys.stdin.readline().rstrip()
ph1,ph2,rlt,tmp=input(),input(),"",[]
for i in range(8):
    rlt+=ph1[i]+ph2[i]
while len(rlt)!=2:
    for i in range(len(rlt)-1):
        tmp+=[str((int(rlt[i])+int(rlt[i+1]))%10)]
    rlt=''.join(tmp)
    tmp=[]
print(rlt)

File: 백준/Bronze/17202. 핸드폰 번호 궁합/핸드폰 번호 궁합.cc
#include <iostream>
using namespace std;
int main(void) {
    string ph1, ph2, rlt = "", tmp = "";
    cin >> ph1 >> ph2;
    for (int i = 0; i < 8; i++) {
        rlt += ph1[i];
        rlt += ph2[i];
    }
    while (rlt.size() != 2) {
        for (int i = 0; i < rlt.size() - 1; i++)
            tmp += to_string(((rlt[i] - 48) + (rlt[i + 1] - 48)) % 10);
        rlt = tmp, tmp = "";
    }
    cout << rlt;
    return 0;
}